rules/hmap.bzl

```diff
@@ -42,17 +42,22 @@ def _make_headermap_impl(ctx):
 
     # Write a file for *this* headermap, this is a temporary file
     input_f = ctx.actions.declare_file(ctx.label.name + "_input.txt")
-    all_hdrs = []
-    for provider in ctx.attr.hdrs:
-        all_hdrs += provider.files.to_list()
+    all_hdrs = list(ctx.files.hdrs)
+    for provider in ctx.attr.direct_hdr_providers:
+        if apple_common.Objc in provider:
```

Comment:
Ah, I assume swift rules provide the `-Swift.h` header in an `objc` provider now no?

Reply:
yeah, they're in `direct_headers`